lib/bots/versionbot.ts

```diff
@@ -352,8 +372,10 @@ export class VersionBot extends ProcBot {
 			});
 
 			// For every one of these, call checkVersioning.
-			return Promise.map(prEvents, (prEvent) => {
-				return this.checkVersioning(registration, prEvent);
+			return Promise.delay(2000).then(() => {
```

Comment:
I've noticed a delay in GH being sent a query and then responding. They send us an event telling us the update has occurred, but then a request to get that update (in this case, a comment on a status failure) doesn't actually get the data back. This improves the chance (and in fact, from testing cures in 'normal' circumstances) that we won't get into a situation where VB comments that a status has failed more than once.